eventually/static/src/containers/register_login/registrationService.js

```diff
@@ -0,0 +1,26 @@
+import axios from "axios"
+
+const apiPath = '/api/v1'
+
+const loginUrl = apiPath + '/user/login/';
+const logoutUrl = apiPath + '/user/logout/';
+const registerUrl = apiPath + '/user/register/';
+const forgetPasswordUrl = apiPath + '/user/forget_password/'
+
+const registerService = (email, password) => {
+    return axios.post(registerUrl, {email, password} );
+}
+
+const loginService = (email, password) => {
+  return axios.post(loginrUrl, {email, password} );
+}
+
+const logoutService = () => {
+  return axios.get(logoutUrl);
+}
+
+const forgetPasswordService = email =>  {
+  return axios.post(registerUrl, {email} );
```

Comment:
should we use there `registerUrl`? because the name of service is `forgetPassword` ? =)

Reply:
done